apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/commons/FDADrugUtilsTest.java

```diff
@@ -8,11 +8,14 @@
 
 public class FDADrugUtilsTest {
 
+  public static final String FAKE_DRUG_CODE_NUMBER = "000000000";
```

Comment:
This should be referenced from the Lookup class where we already have a constant (and I believe this has a difference value btw and should not but let me know if that is incorrect).